src/commands/metadata-templates/update.js

```diff
@@ -301,6 +311,10 @@ MetadataTemplatesUpdateCommand.flags = {
 		description: 'Add a multi-select field with the provided display name',
 		multiple: true,
 	}),
+	'add-multi-select-option': flags.string({
+		description: 'Add an option to a specified multiselect field; must be followed by one or more --option flags',
```

Comment:
```suggestion
		description: 'Add an option to a specified multiselect field; must be followed by one or more --multi-select-option flags',
```

Reply:
I think `option` might be too generic. Is this OK?

Reply:
How are these `multi-select-option` flags tied to the usage of add-multi-select-option?  How does it work when there are multiple `multi-select-option` flags?